core/schema_utils_test.py

```diff
@@ -220,8 +222,8 @@
 }
 
 
-# The type of `validator` is Dict[str, Any] here because its values represent
-# objects for normalization, and they can have any type.
+# Here we use type Any because the argument `validator` represent objects
```

Comment:
... represents the object to be normalized, and that object can ...

Reply:
Done